baram/ec2_manager.py

```diff
@@ -1,105 +1,290 @@
 import boto3
+import traceback
+
+from baram.log_manager import LogManager
 
 
 class EC2Manager(object):
     def __init__(self):
         self.cli = boto3.client('ec2')
+        self.logger = LogManager.get_logger()
 
     def list_security_groups(self):
-        '''
+        """
         Describes the specified security groups or all of your security groups.
 
         :return: SecurityGroups
-        '''
+        """
         return self.cli.describe_security_groups()['SecurityGroups']
 
+    def list_redundant_security_group_ids(self, redundant_domain_ids: list = None):
```

Comment:
According to your logic, I think `list_unused_sg_ids` is more appropriate.

Reply:
Thanks for your opinion.